examples/with-sentry/pages/_error.js

```diff
@@ -1,56 +1,82 @@
-import NextErrorComponent from 'next/error'
+/**
+ * NOTE: If you are on `@sentry/nextjs` version 7.2.1 or higher, you can replace
+ * what's below with the simpler version which appears in a comment at the end
+ * of this file.
+ *
+ * This page is loaded by Nextjs:
+ *  - on the server, when data-fetching methods throw or reject
+ *  - on the client, when `getInitialProps` throws or rejects
+ *  - on the client, when a React lifecycle method throws or rejects, and it's
+ *    caught by the built-in Nextjs error boundary
+ *
+ * See:
+ *  - https://nextjs.org/docs/basic-features/data-fetching/overview
+ *  - https://nextjs.org/docs/api-reference/data-fetching/get-initial-props
+ *  - https://reactjs.org/docs/error-boundaries.html
+ */
 
 import * as Sentry from '@sentry/nextjs'
+import NextErrorComponent from 'next/error'
 
-const MyError = ({ statusCode, hasGetInitialPropsRun, err }) => {
-  if (!hasGetInitialPropsRun && err) {
-    // getInitialProps is not called in case of
-    // https://github.com/vercel/next.js/issues/8592. As a workaround, we pass
-    // err via _app.js so it can be captured
-    Sentry.captureException(err)
-    // Flushing is not required in this case as it only happens on the client
+const COMPATIBILITY_ERROR =
+  '[Sentry] Please update `@sentry/nextjs` to at least version 7.2.1 to use ' +
+  "`captureUnderscoreErrorException`, or if you can't upgrade, use the " +
+  'older version of this page found at ' +
+  'https://github.com/getsentry/sentry-wizard/blob/v1.2.17/scripts/NextJs/configs/_error.js.'
+
+const CustomErrorComponent = (props) => {
+  if (typeof Sentry.captureUnderscoreErrorException === 'function') {
```

Comment:
This example app is already using the v7+ (newest) version and our example should be reflecting the version in package.json of this example, not all versions of our SDK. With this in mind, I don't think that we need this. We can just call `Sentry.captureUnderscoreErrorException(props)` without checking for `if (typeof Sentry.captureUnderscoreErrorException === 'function')...`

Reply:
> This example app will be using v7.2.1 and our example should be reflecting the version in package.json of this example, not all versions of our SDK.

That's fair. I copied this over from our wizard, where we can't guarantee what nextjs version they'll be using, but you're right that in this case we actually can. That will simplify this quite a bit (and handle your concern above about the error message, which in the wizard isn't included in the function because most people should be able to delete it and save the bundle size, but which here becomes a moot point).